crates/types/src/logs/metadata.rs

```diff
@@ -222,27 +229,36 @@ impl Chain {
         // NOTE: Hopefully at some point we will use the nightly Cursor API for
```

Comment:
Rustdoc for `find_segment_for_lsn` seems outdated.

Reply:
Is it correct now?

Reply:
👍